src/main/java/com/lambdaschool/wellness/controllers/GroupController.java

```diff
@@ -16,7 +16,7 @@
 
 @RestController
 @RequestMapping(value = "/api/group")
-@CrossOrigin("http:localhost:3000")
+@CrossOrigin(origins = "http://localhost:3000", maxAge = 3600)
```

Comment:
Does this need to be here? You can do the CrossOrigin configuration in the WebConfig.java
In Intellij you can set environment variables in the program itself.